resources/skins/nudus/html_wrapper.html

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE html>
+<html lang="en">
+<!--
+  ___________________________
+< I'm an expert in my field. >
+  ---------------------------
+         \   ^__^
+          \  (oo)\_______
+             (__)\       )\/\
+                 ||----w |
+                 ||     ||
+-->
+<head>

```

Comment:
Also, I prefer to use 2 spaces per indendation level, maybe it would be better to keep things consistent across all skins?

Reply:
I am not sure if I would do that. I always do a four space...

I would, only if you insist.

Reply:
no problem then